containers/events/events.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
This file feels awfully excessive. As well as the `EventsTagline` component. Do we really need to make everything a component (which potentially introduces excess `div` s in the markup, which is **not acceptable**)? IMHO, the only thing that should really be a component is something that is data-dependent (event cards) or has logic integrated (filters block). The header and tagline are not deserving of a component to me. And if that helps fight excess markup, by all means, go for it

Reply:
Components do not introduce an extra `div`. They get replaced by the template's root element. They help with abstraction and avoiding repetition. Sometimes their main goal is to simply isolate a block that has an independent purpose from an otherwise very lengthy code.

Reply:
Yeah, it's just that sometimes a component may not have a root element, which forces to create container elements which weren't part of the markup. It's not a hit at this particular code piece, I'd just like to express that if we are picking between clean markup and short files, we should probably go for clean markup

Reply:
You mean not have a template at all? I don't think a component like that would be useful. In case you meant multiple root elements, this is not even allowed in Vue. Templates can only have one root element.

Reply:
What I mean is, consider, for example, a component that consists of two sibling elements. The only way to make it a Vue component is to introduce an outer tag to wrap those two